### PR TITLE
error handling

### PR DESCRIPTION
**Pull request by egmalik**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -9,9 +9,9 @@
     <string name="location">Location</string>
     <string name="feeling">Feeling</string>
     <string name="this_was_at">This was at</string>
-    <string name="month">mm</string>
-    <string name="year">yyyy</string>
-    <string name="day">dd</string>
+    <string name="month">month</string>
+    <string name="year">year</string>
```
Comment (lielfr) [inline app/src/main/res/values/strings.xml]: This is wrong. You should make sure you still have the MM,YYYY,DD strings and create new strings with different names.

**Review by lielfr — CHANGES_REQUESTED** [review]
Pay attention to the strings.xml file.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (egmalik) [reply app/src/main/res/values/strings.xml]: ✅ correcting the strings

File: app/build.gradle [hunk 2]
```diff
@@ -47,13 +47,21 @@ dependencies {
...
     implementation 'com.borjabravo:readmoretextview:2.1.0'
     implementation 'com.google.code.gson:gson:2.8.5'
     implementation 'com.squareup.retrofit2:retrofit:2.6.1'
     implementation 'com.squareup.retrofit2:converter-gson:2.6.1'
     implementation 'com.google.firebase:firebase-storage:9.2.1'
     implementation 'com.google.firebase:firebase-auth:9.2.1'
 
+    implementation 'com.squareup.retrofit2:converter-gson:2.6.1'
+    implementation 'com.squareup.retrofit2:retrofit:2.6.1'
+    implementation 'com.github.bumptech.glide:glide:4.9.0'
```
Comment (lielfr) [inline app/build.gradle]: Also here - pay attention to the previous lines, you already have what's necessary.

**Pull request merged**